### ISSUE TITLE
confiugre file for podnet_nme_cifar100

### ISSUE THREAD

**Issue by kminsoo** [arthurdouillard/incremental_learning.pytorch#31]
I run your code using podnet_nme_cifar100 & podnet_cnn_cifar100.

In case of podnet_cnn_cifar100, I got nearly same results as yours reported in your paper. 
However, I could not get your results for podnet_nme_cifar100(mine: 56.78/0.41, yours: 61.40/0.68). 
Q1), Could you check your podnet_nme_cifar100.yaml ? 

Also, I would like to ask another question about how many gpus you use for ImageNet experiment?
In the configure file for ImageNet experiments, you use a batch size of 64.
Q2) However, in your main paper, the batch size is set to 128 for all datasets. Could you let me know about the batch size and the number of gpus for ImageNet ?

**Comment by kminsoo** [comment]
I would like to ask another question about how many gpus you  use for ImageNet experiment? 
In the configure file for ImageNet experiments, you use a batch size of 64. 
Does it imply that you use a total batch size of 64 * n (the number of gpus)?

**Comment by arthurdouillard** [comment]
Hello,

1. I have, yet, no idea why but it seems that the NME branch is working well on torch==1.2.0 but not on later version (I've tested 1.7.0). I got 60.39 ± 0.73 with PODNet NME 50 + 50 x 1 on 3 seeds (1, 2, 3) with torch==1.2.0 and command:

```
python3 -minclearn --options options/podnet/podnet_nme_cifar100.yaml options/data/cifar100_3orders.yaml --initial-increment 50 --increment 1 --device 0 --autolabel
```

I've run this yesterday using master code, so I hope it will work for you.

2. I've used 2 GPUs for ImageNet, I've tried both 128 and 64 batch_size for ImageNet. Results are similar as long as you scale the learning rate appropriately. Meaning that given a provided config file with batch_size=64 and lr=0.05, you can try batch_size=128 and lr=0.1

I hope it answers your questions :)

**Comment by kminsoo** [comment]
Thank you for your reply!

**Comment by eddielyc** [comment]
Hi, sorry for reopening this issue, but I came across the same problem with PyTorch 1.7 and tried to reproduce the PODNet NME with PyTorch 1.2.  With the command [here](https://github.com/arthurdouillard/incremental_learning.pytorch/issues/31#issuecomment-766335321), I successfully got the similar results 60.41 +/- 0.86. However, we find this in the log file:

> 2021-07-12:11:02:42 [podnet.py]: Now 40 examplars per class.

which was printed before first incremental task and I believe the ablation studies followed the fixed memory protocol. In the command above, fixed-memory option is missing and 
`self._fixed_memory = args.get("fixed_memory", True)` might fails to run as expected because if fixed_memory is missing in command, parser would automatically set fixed_memory=False. After manually set fixed_memory=True, PODNet still got 56.91 +/- 0.6. I wonder if there is something wrong with my config or something else to improve the results. 

Thanks :)
